CHANGELOG.md

```diff
@@ -89,6 +98,17 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 * [#758](https://github.com/allora-network/allora-chain/pull/758): Ensure pending stake removals are processed
 * [#767](https://github.com/allora-network/allora-chain/pull/767) Fix [ISA-2025-001](https://github.com/cosmos/ibc-go/security/advisories/GHSA-4wf3-5qj9-368v)
 
+### API Breaking Changes
+
+#### Removed
+
+* GetLatestAvailableNetworkInferences()
+* GetLatestAvailableNetworkInferencesOutlierResistant()
```

Comment:
```suggestion
* [#769](https://github.com/allora-network/allora-chain/pull/769): `x/emissions` gRPC query `GetLatestAvailableNetworkInferences()`, use `GetLatestNetworkInferences()` instead
* [#769](https://github.com/allora-network/allora-chain/pull/769): `x/emissions` gRPC query `GetLatestAvailableNetworkInferencesOutlierResistant()` , use `GetLatestNetworkInferencesOutlierResistant()` instead
```

We could be more specific and indicate that it concerns the gRPC API of the `x/emissions` module, and I think it should be fine to reference the same PR multiple times